### ISSUE TITLE
Playlist icons not always loading

### ISSUE THREAD

**Issue by KoljaWindeler** [pathofleastresistor/polr-ytube-media-card#1]
Hi @pathofleastresistor I absolutely love what you are doing here.
Just tested 1.5
Small ideas:
- if we only one track is in the playlist it looks odd. Can the card height be adjusted dynamically? 
- could you show the covers when searching ?

Again: greet work!

**Comment by pathofleastresistor** [comment]
> if we only one track is in the playlist it looks odd. Can the card height be adjusted dynamically?

Yeah I agree that a single track does look weird right now. That said, I explicitly chose to make the card a fixed height to avoid issues where dynamic card height changes cause weird layout issues (especially if you're doing a masonry layout). I'll think more about better layouts though.

> could you show the covers when searching ?

Perhaps I'm missing something, but I do see covers when searching. More generally speaking, if the item has a thumbnail set in the Media Browser, this card should prefer to show it over the basic music note icon.

<img width="511" alt="image" src="https://github.com/pathofleastresistor/polr-ytube-media-card/assets/91756648/f3040572-fb25-4bd7-bfa2-23294aa36920">

**Comment by KoljaWindeler** [comment]
ha .. you're right .. during search there are icons .. but not in the playlist view
![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/471314/45ce9a89-b70d-491a-8e9f-8da24d93257a)

where do you get the "next items" list from anyway? from the extra sensor? 

understand the fixed layout argument .. fair enough

**Comment by pathofleastresistor** [comment]
Ahh okay - so right now I get the "next items" from "Current Playlists" in the Media Browser. I checked in the media browser and that also doesn't show album art. All this card does is send a WebService call to get the data from that page.

I think your component sets it here - https://github.com/KoljaWindeler/ytube_music_player/blob/main/custom_components/ytube_music_player/browse_media.py#L496

**Comment by KoljaWindeler** [comment]
Funny .. it works with playlists.  
Ok so not a to-do for you but for me 
Excellent. 
https://github.com/KoljaWindeler/ytube_music_player/issues/286

**Comment by KoljaWindeler** [comment]
Works fine with https://github.com/KoljaWindeler/ytube_music_player/commit/930b590d007aeb9a744953131c10162238f803d9

**Comment by KoljaWindeler** [comment]
ok, i don't get it. I've added a call to copy the thumbnail from an album to each track
and confirmed that it is working with the current_playlist in the media_browser
![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/471314/ac89ad49-9c84-469e-85f8-fc5bf262759d)

works .. but on the card it is not visible .. strange right? 

![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/471314/5f526e5d-b4bd-4fa0-b932-4585d7cf54a5)
